### PR TITLE
ZIntBitInputVectorTest and CsvTupleParserTest

### PR DESCRIPTION
**Pull request by qlinsey**
add ZintBitInputVector test and CsvTupleParserTest ( not sure whether CsvTupleParser should be removed or not)

### CODE REVIEW

File: cool-core/src/test/java/com/nus/cool/core/io/storevector/ZIntBitInputVectorTest.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import com.nus.cool.core.util.ArrayUtil;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+public class ZIntBitInputVectorTest{
+    @Test(dataProvider = "ZIntBitDP")
```
Comment (Zrealshadow) [inline cool-core/src/test/java/com/nus/cool/core/io/storevector/ZIntBitInputVectorTest.java:19]: Change the UnitTest Method to `ZIntBitInputVectorUnitTest` not `testRead`

File: cool-core/src/test/java/com/nus/cool/core/resources/parsertest/table.csv [hunk 2]
```diff
@@ -0,0 +1,4 @@
+id,birthYear,event,attr1,attr2,attr3,attr4,time
```
Comment (Zrealshadow) [inline cool-core/src/test/java/com/nus/cool/core/resources/parsertest/table.csv:1]: I think you can use the csv table which we have created before.
No need to create another mini dataset

File: cool-core/src/test/java/com/nus/cool/core/util/parser/CsvTupleParserTest.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                "core",
+                "resources").toString();
+    }
+
+    @Test
+    public void parseTest(){
+        String filepath = Paths.get(sourcePath, "parsertest", "table.csv").toString();
+        System.out.println(filepath);
+        CsvTupleParser csvParser = new CsvTupleParser();
+        try {
```
Comment (Zrealshadow) [inline cool-core/src/test/java/com/nus/cool/core/util/parser/CsvTupleParserTest.java:37]: Personally, directly throw exception in unitTest is better. No need to catch expection in unitTest

File: cool-core/src/test/java/com/nus/cool/core/io/storevector/ZIntBitInputVectorTest.java (hunk 1 above)
Comment (qlinsey) [reply cool-core/src/test/java/com/nus/cool/core/io/storevector/ZIntBitInputVectorTest.java:19]: Thanks, will update.

File: cool-core/src/test/java/com/nus/cool/core/resources/parsertest/table.csv (hunk 2 above)
Comment (qlinsey) [reply cool-core/src/test/java/com/nus/cool/core/resources/parsertest/table.csv:1]: agree, thanks

File: cool-core/src/test/java/com/nus/cool/core/util/parser/CsvTupleParserTest.java (hunk 3 above)
Comment (qlinsey) [reply cool-core/src/test/java/com/nus/cool/core/util/parser/CsvTupleParserTest.java:37]: sure, thanks.

**Comment by qlinsey** [comment]
updated, thanks.

**Review by Zrealshadow — COMMENTED** [review]
Please add newline in the end of  these two new committed files.
Any files should be ended with \n.

**Comment by qlinsey** [comment]
thanks, i think i added newline. I saw the red circle gone.

**Review by KimballCai — COMMENTED** [review]
Nice work.
I have checked the code, and it works properly. The code format is also appropriate.

**Review by Zrealshadow — APPROVED** [review]

**Comment by qlinsey** [comment]
Thanks!

**Pull request merged**